src/main/kotlin/io/perfometer/dsl/HttpDsl.kt

```diff
@@ -41,6 +44,11 @@ class RequestBuilder(
         return this
     }
 
+    fun process(process: (response: HttpResponse) -> Unit): RequestBuilder {
```

Comment:
Maybe `fun process(processor: (HttpResponse) -> Unit): RequestBuilder`?

Note that it's not necessary to include parameter name in functional type definitions, so `(response: HttpResponse) -> Unit` can be shortened to `(HttpResponse) -> Unit`

Reply:
thanks for the tip